Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM codeclimate/alpine-ruby:b38
+FROM ruby:2.3-alpine
```

Comment:
Current is 69.7 MiB. The new one is 165.9 MiB. So yes, increase of almost 100 MiB.
What is the threshold?